src/contracts/core/AVSDirectoryStorage.sol

```diff
@@ -30,17 +35,30 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
     mapping(address => mapping(address => OperatorAVSRegistrationStatus)) public avsOperatorStatus;
 
     /// @notice Mapping: operator => 32-byte salt => whether or not the salt has already been used by the operator.
-    /// @dev Salt is used in the `registerOperatorToAVS` function.
+    /// @dev Salt is used in the `registerOperatorToAVS` and `registerOperatorToOperatorSet` function.
     mapping(address => mapping(bytes32 => bool)) public operatorSaltIsSpent;
 
-    constructor(IDelegationManager _delegation) {
+    /// @notice Mapping: AVS => whether or not the AVS uses operator set
+    mapping(address => bool) public isOperatorSetAVS;
+
+    /// @notice Mapping: AVS => operatorSetID => strategy => whether or not the strategy is in the operator set
+    mapping(address => mapping(uint32 => mapping(IStrategy => bool))) public operatorSetStrategies;
```

Comment:
what does "in the operator set" mean? That they can slash it? or that they are paying for it?

Reply:
They can eventually slash it if the operator has also set its strategy in the operator set to slashable. Mainly used for the UI thought to properly account for strategies in each operator set.